src/sampling.zig

```diff
@@ -902,7 +915,10 @@ pub fn retimed_linear_curve_interpolating(
     return Sampling{
         .allocator = allocator,
         .buffer = full_output_buffer,
-        .sample_rate_hz = in_samples.sample_rate_hz,
+        // @TODO: I remember one conversation about splitting
```

Comment:
was that to accommodate the case where libsamplerate doesn't want to give back the exact number of samples we require?